java/CMakeLists.txt

```diff
@@ -547,3 +668,20 @@ if(NOT MINGW)
     COMPILE_PDB_NAME ${ROCKSDBJNI_STATIC_LIB}.pdb
   )
 endif()
+
+if(WITH_LZ4)
+#  find_package(lz4 REQUIRED)
+#  add_definitions(-DLZ4)
+  include_directories(${lz4_INCLUDE_DIRS})
+#  list(APPEND THIRDPARTY_LIBS lz4::lz4)
+endif()
```

Comment:
Isn't this already being handled when we build the base RocksDB library?

Reply:
I did a build with this block commented out, and everything built correctly: https://github.com/sightmachine/rocksdb/commit/4c73a5819948ca7f7d1ad78d58cb153066739ca9

I think we can remove this.